app/test_main.py

```diff
@@ -1,3 +1,24 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+@pytest.mark.parametrize(
+    "cat_age,dog_age,result",
+    [
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (50, 50, [8, 7]),
+        (-15, 14.5, [0, 0]),
+        (23, 25, [1, 2]),
+        (27, 25, [2, 2]),
+    ],
+    ids=[
+        "14 cat/dog years -> 0 human age",
+        "15 cat/dog years -> 1 human age",
+        "50 cat years -> 6 human, 50 dog years -> 5 human",
```

Comment:
The description for the test case `"50 cat years -> 6 human, 50 dog years -> 5 human"` is incorrect. It should be `"50 cat years -> 8 human, 50 dog years -> 7 human"` based on the conversion rules provided in the task description.